pkgs/tools/security/sipvicious/default.nix

```diff
@@ -1,26 +1,46 @@
 { lib
-, buildPythonApplication
 , fetchFromGitHub
+, installShellFiles
+, python3
 }:
 
-buildPythonApplication rec {
+python3.pkgs.buildPythonApplication rec {
```

Comment:
What is the difference using `python3.pkgs` vs `python3Packages` here? And when the use of one is preferred over the other?

Reply:
Using `python3.pkgs` allows to override dependencies. Also, in the long run it will help to get of `python3Packages.callPackage` in `all-packages.nix`. Could maybe become a requirement for the automatic transition to `by-name` which will happen at some time in the future..